src/retro_data_structures/formats/pak_wii.py

```diff
@@ -38,10 +48,12 @@ def _emitparse_header(code: construct.CodeGen) -> str:
 ConstructResourceHeader._emitparse = _emitparse_header
 
 PAKNoData = Struct(
-    _start=construct.Tell,
+    _start=construct.Tell,      ##Should always be 0x00
     _header=PAKHeader,
+    _table_of_contents_start=construct.Tell,        ##Should always be 0x40
```

Comment:
Consider making comments like this instead:
```suggestion
    _table_of_contents_start=construct.Tell,        # Should always be 0x40
```

One `#`, then a space, then starting with an uppercase character like it's the start of a sentence.
(Last one sometimes gets forgotten by us, but it reads slightly nicer and IIRC pycharm yells at you if you start lowercase)